kite-data/kite-data-crunch/src/main/java/org/kitesdk/data/crunch/CrunchDatasets.java

```diff
@@ -96,6 +96,52 @@
   }
 
   /**
+   * Expose the given {@link View} as a Crunch {@link ReadableSource}.
+   *
+   * @param view the view to read from
+   * @param readerSchema the schema used to read from the view
+   * @param <E>     the type of entity produced by the source
+   * @return a {@link ReadableSource} for the view
+   *
+   * @since 1.1.0
+   */
+  public static <E> ReadableSource<E> asSource(View<E> view, Schema readerSchema) {
```

Comment:
Do you think it's worth adding these APIs now just for this issue, or would it make sense to give the CopyTask a private API to solve the current problem and add more general support for this in a follow-up?